src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const objKeys = Object.keys(robot);
+  const objValues = Object.values(robot);
+  const trueRobot = {};
+
+  for (let i = 0; i < objKeys.length; i++) {
```

Comment:
you can also use `Object.entries()` to make code more simple https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries